src/main/webapp/content/icons/icons.ts

```diff
@@ -42,8 +42,23 @@ export const facSaveWarning: IconDefinition = {
     ],
 } as IconDefinition;
 
+export const facDetails: IconDefinition = {
```

Comment:
Can you link the source here?

Reply:
this is a custom icon from @rabeatwork so i dont think this is necessary